lib/CloudConvert.ts

```diff
@@ -70,7 +70,10 @@ export default class CloudConvert {
                     ? 'https://socketio.sandbox.cloudconvert.com'
                     : 'https://socketio.cloudconvert.com',
                 {
-                    transports: ['websocket']
+                    transports: ['websocket'],
+                    reconnection: true,
+                    reconnectionDelay: 2 * 60 * 1000, // two minutes
+                    reconnectionDelayMax: 15 * 60 * 1000 // 15 minutes
```

Comment:
Why do the defaults need to be increased like this?
Defaults:
https://github.com/socketio/socket.io/blob/main/packages/socket.io-client/lib/manager.ts#L55